learn/advanced/filtering_and_faceted_search.md

```diff
@@ -316,6 +316,10 @@ Using the `facetsDistribution` search parameter adds two new keys to the returne
 `exhaustiveFacetsCount` is not currently implemented in and will always return `false`.
 :::
 
+::: note
+By default, `facetDistribution` returns a maximum of 100 facet values for each faceted field. You can change this value using the `maxValuesPerFacet` property of the [`faceting` index settings](/reference/api/faceting.md).
```

Comment:
```suggestion
By default, `facets` returns a maximum of 100 facet values for each faceted field. You can change this value using the `maxValuesPerFacet` property of the [`faceting` index settings](/reference/api/faceting.md).
```
`facetsDistribution` is now called [`facets`](https://github.com/meilisearch/documentation/pull/1734)

Reply:
hmmmm…that's interesting. In my head I was referring to the response value, which is now `facetDistribution`, but I think you're right, it makes more sense to refer to the request parameter.